portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,18 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+// Automatic Slideshow - change image every 3 seconds
+var myIndex = 0;
+carousel();
+
+function carousel() {
```

Comment:
Try to add a verb into your function names to make it clear what it's doing to the carousel
https://google.github.io/styleguide/jsguide.html#naming-method-names

Reply:
fixed - new name is showSlides